Apps/WebClient/src/ClientApp/src/models/immunizationTimelineEntry.ts

```diff
@@ -16,27 +16,25 @@ export default class ImmunizationTimelineEntry extends TimelineEntry {
         }
 
         let text =
-            (this.immunization.name! || "") +
-            (this.immunization.agents.toString() || "");
+            (this.immunization.location! || "") +
+            (this.immunization.type! || "");
         text = text.toUpperCase();
         return text.includes(filterText.toUpperCase());
     }
 }
 
 class ImmunizationViewModel {
-    public name: string;
-    public status?: string;
-    public agents: string;
-    public occurrenceDateTime: Date;
+    public id: string;
+    public isSelfReported: boolean;
+    public location: string;
+    public type: string;
+    public immunized: Date;
```

Comment:
dateOfImmunization or something along those lines

Reply:
how about immunizedOn for a shorter name?